Packs/RSANetWitness_v11_5/Integrations/RSANetWitness115/RSANetWitness115.yml

```
@@ -0,0 +1,1401 @@
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    deprecated: false
+    description: initiate the download of the system dump to the Endpoint Server.
+    execution: false
+    name: rsa-nw-system-dump-download-request
+  - arguments:
+    - default: false
+      description: Agent ID of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    - default: false
+      description: ID of the process.
+      isArray: false
+      name: process_id
+      required: false
+      secret: false
+    - default: false
+      description: Identifier of the process in windows.
+      isArray: false
+      name: eprocess
+      required: true
+      secret: false
+    deprecated: false
+    description: initiate the download of the process dump to the Endpoint Server.
+    execution: false
+    name: rsa-nw-process-dump-download-request
+  - arguments:
+    - default: false
+      description: Unique identifier of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      name: allow_dns_only_by_system
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - default: false
+      description: IP addresses excluded from isolation, seperated by commas adresses
+        can be v4 or v6. example - 1.2.3.4,11:22:33:44
+      isArray: false
+      name: exclusion_list
+      required: false
+      secret: false
+    - default: false
+      description: additional information
+      isArray: false
+      name: comment
+      required: true
+      secret: false
+    deprecated: false
+    description: Isolates the host with the specified agent ID from the network.
+    execution: false
+    name: rsa-nw-endpoint-isolate-from-network
+  - arguments:
+    - default: false
+      description: Unique identifier of the host.
+      isArray: false
+      name: agent_id
+      required: true
+      secret: false
+    - default: false
+      description: Service ID of the Endpoint Server to be connected.
+      isArray: false
+      name: service_id
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: allow_dns_only_by_system
+      required: false
+      secret: false
+    - default: false
+      description: ' IP addresses excluded from isolation, seperated by commas adresses
+        can be v4 or v6. example - 1.2.3.4,11:22:33:44'
+      isArray: false
+      name: exclusion_list
+      required: false
```

Shouldn't this be required as this is the sole purpose of this command?
```suggestion
      required: true
```

---

Yep, changed it :)